apps/src/StudioApp.js

```diff
@@ -1579,8 +1586,8 @@ StudioApp.prototype.handleHideSource_ = function (options) {
       $('body').append(wireframeSendToPhone);
     }
     document.body.style.backgroundColor = '#202B34';
-  // For share page on mobile, do not show this part.
-  } else if (!options.embed && !(this.share && dom.isMobile())) {
+  // For share page, do not show this part.
+  } else if (!options.embed && !this.share) {
```

Comment:
Hmmm....not too sure, as it's really across everything. Maybe CS Fundamentals? Developers kind of makes the most sense since this is tech debt, but I don't think that board is really monitored in any sort of way. I'm also kind of okay with not opening an item here. Maybe the next person to spend time in this code sees the TODO and cleans it up, or worst case we have some dead code.